benchmark_runner/common/template_operations/template_operations.py

```diff
@@ -125,6 +128,7 @@ def generate_yamls_internal(self, scale: str = None, scale_node: str = None, red
             'standard_output_file': self.__standard_output_file,
             'standard_template_file': self.__standard_template_file,
             'scale': scale,
+            'scale_num': scale_num,
```

Comment:
This is where `scale` and `scale_num` or whatever you want to call it should be converted from numbers to strings.